src/components/DayBoxSchedule.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from "react";
+import List, { ListItem } from "material-ui/List";
+import Typography from "material-ui/Typography";
+class DayBoxSchedule extends Component {
+  displayClasses() {
+    let card = [];
+    let divKey = 0;
+    if (
+      this.props.calendarMeeting[this.props.month] === undefined ||
+      this.props.calendarMeeting[this.props.month][this.props.day] === undefined
+    ) {
+      ++divKey;
+      card.push(<div key={"noclasses" + divKey} aria-label="No classes" />);
+    } else {
+      if (
+        this.props.calendarMeeting[this.props.month][this.props.day] !==
+        undefined
+      ) {
+        if (
+          this.props.calendarMeeting[this.props.month][this.props.day].length >
+          2
+        ) {
+          let moreClasses = 0;
+          moreClasses =
+            this.props.calendarMeeting[this.props.month][this.props.day]
+              .length - 2;
+          for (let i = 0; i < 2; i++) {
+            card.push(
+              <ListItem
+                component="div"
+                key={"classSchedule" + i}
+                style={{
```

Comment:
JSS stylesheets will make this file a little shorter, since the ListItems in the different conditional statements share the same CSS :)